src/patterns/names/index.md.njk

```diff
@@ -51,7 +51,7 @@ If users are from outside the UK, use the labels:
 
 Make middle names optional.
 
-Make it clear whether you need someone’s common name or their full legal name.
+Make it clear whether you need someone’s common name, or if it needs to match their name as used on official documents such as a passport or driving licence.
```

Comment:
Looks good to me, just suggesting a small tweak to shorten the sentence slightly. 

```suggestion
Make it clear whether you need someone’s common name, or their name as it's written on official documents such as a passport or driving licence.
```